ofono/include/sms-filter.h

```diff
@@ -153,6 +153,7 @@ struct ofono_sms_filter {
 				const struct ofono_sms_address *addr,
 				int dst_port, int src_port,
 				const unsigned char *buf, unsigned int len,
+				int flags,
```

Comment:
This is an interface too.

Reply:
Removed with a31e99b0a6bb56b996c9d1e9f3bd7cf0aef2f5a0